docker/Dockerfile-dev

```diff
@@ -53,13 +53,12 @@ RUN apt-get update \
     # Install Go tools.
     && bash /tmp/staging/install-go-tools.sh "${USERNAME}" \
     #
-    # Install protoc compiler.
-    && LATEST_PROTOC_URL=$(curl -s https://api.github.com/repos/protocolbuffers/protobuf/releases/latest | grep -oP '"browser_download_url": "\K(.*)(?=")' | grep "protoc-.*-linux-x86_64") \
-    && wget -q -O /tmp/staging/protoc-latest.zip ${LATEST_PROTOC_URL} \
-    && unzip /tmp/staging/protoc-latest.zip -d /tmp/staging/protoc \
-    && chmod -R 755 /tmp/staging/protoc \
-    && mv /tmp/staging/protoc/include/* /usr/local/include \
-    && mv /tmp/staging/protoc/bin/* /usr/local/bin \
+    # Install protoc compiler 3.14.0 currently required by 'make gen-proto'.
+    && PROTOC_ZIP=protoc-3.14.0-linux-x86_64.zip \
```

Comment:
Nit: suggest keeping the version in its own variable.

Reply:
Since you are planning to update it anyway for Darwin ARM64 native feel free to make the change then. We never update the protoc version so far, so this isn't necessary :)